README.md

```diff
@@ -91,7 +91,8 @@ git rebase -i **<TAB>
 - systemctl
 - yarn
   - run
-  - workspace
+  - workspace (requires [jq][])
+  - workspace <workspace> run (requires [jq][])
```

Comment:
* fzf-zsh-completions depends on jq in kubectl as well and it looks cumbersome to write dependencies everywhere. How about simply move this to prerequisites?
* Do not forget to escape in Markdown as it must be valid HTML.

```suggestion
  - workspace
  - workspace \<workspace\> run
```